mappings/translation_test_utilities.go

```diff
@@ -0,0 +1,345 @@
+package mappings
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+)
+
+// the tag db is what is used to convert a struct to it's relevant DB field
+const tagDBName = "db"
+
+// baseStructExcludeFields these are shared excluded fields that don't need to have a translation
+var baseStructExcludeFields []string = []string{"ID", "CreatedBy", "CreatedDts", "ModifiedBy", "ModifiedDts"}
+
+// taskListStructExcludeFields are the fields that aren't needed to be translated for most task list sections
+var taskListStructExcludeFields []string = append(baseStructExcludeFields, "ModelPlanID")
```

Comment:
Better as a map for O(1) lookup times. Ideally would be a hash set but not natively supported by Go. Can do a map[string]bool store for simple key lookups.

Reply:
Thanks @OddTomBrooks, I"m taking a middle ground approach on this. I'm leaving the slices for the variable, as they make more sense when you look at them, and allow append etc. I'm converting the slice to a map in the assertTranslationStructCoverage function. This adds a little to that function, but is a net complexity reduction while maintaining readability. I think that should be sufficient for these tests.